challenge3/webapp/caesaris/app/models/dataset.rb

```diff
@@ -0,0 +1,15 @@
+class Dataset < ApplicationRecord
+  belongs_to :house
+  delegate :city, to: :house
```

Comment:
```suggestion
  has_one :city, through: :house
```
has_one を使うことで、 delegate と違って、各種アソシエーション関連の
機能を使えるようになる。

Reply:
そうですね、私もつい最近まで`belongs_to, through`を実装したい時、`delegate`を使うと信じ込んでいましたが、確かに`has_one`でDIするとRansackもすらすらかけるとつい最近気づきました。